src/components/Dropdown/V2/__tests__/Dropdown.Item.test.tsx

```diff
@@ -157,6 +202,33 @@ describe('Events', () => {
     expect(spy).toHaveBeenCalled()
   })
 
+  test('onClick callback fires (selectionClearer)', () => {
+    const spy = jest.fn()
+    const state = {
+      allowMultipleSelection: true,
+      selectionClearer: 'All Items',
+    }
+    const wrapper = mount(<Item onClick={spy} getState={() => state} />)
+
+    wrapper.simulate('click')
+
+    expect(spy).toHaveBeenCalledWith(
+      state,
+      expect.objectContaining({
```

Comment:
@tinkertrain Whoa! I've never seen this before 😮

Reply:
Me neither to be honest! I wanted to check that this function was called with the correct state, and didn't care about the shape of `Event`, there's no "partial" `toHaveBeenCalledWith` but I found this, which allows you to just check that an object has a certain field, I just chose one that is present in `Event` and voila!